plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/JfrogArtifactoryRepository.tsx

```diff
@@ -80,11 +80,8 @@ export function JfrogArtifactoryRepository(props: RepositoryProps) {
   );
 }
 
-JfrogArtifactoryRepository.defaultProps = {
-  title: 'Docker Images',
-};
 interface RepositoryProps {
   widget: boolean;
   image: string;
-  title: string;
+  title?: string;
```

Comment:
as the component needs only an image, so can this type of definition just have that?

Reply:
Here i see widget and image is used in JfrogArtifactoryDashboardPage: https://github.com/janus-idp/backstage-plugins/blob/359d7da547651a8686b4f2b754ec11f0b75b6829/plugins/jfrog-artifactory/src/components/JfrogArtifactoryDashboardPage/JfrogArtifactoryDashboardPage.tsx#L12C11-L12C37

I think we cannot remove widget but as per my understanding we can remove the title from definition.

Reply:
yeh, it is passed but not used in component `JfrogArtifactoryRepository` so can be removed from https://github.com/janus-idp/backstage-plugins/blob/359d7da547651a8686b4f2b754ec11f0b75b6829/plugins/jfrog-artifactory/src/components/JfrogArtifactoryDashboardPage/JfrogArtifactoryDashboardPage.tsx#L12C11-L12C37 and then from types as well.

Reply:
Sure, i will make the suggested changes.